error/src/lib.rs

```diff
@@ -30,7 +30,7 @@ pub struct Error {
 impl fmt::Display for Error {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
         if let Some(cause) = self.cause() {
-            write!(f, "{}({})", self.kind(), cause)
+            write!(f, "{}: {}", self.kind(), cause)
```

Comment:
If you wanna change the style of error messages, please change other error types as well:

```
$ agg 'impl fmt::Display for' | grep Error
2:     2	test/src/rpc/error.rs:9:impl fmt::Display for Error {
10:    10	spec/src/lib.rs:175:impl fmt::Display for SpecLoadError {
11:    11	verification/src/error.rs:237:impl fmt::Display for HeaderError {
12:    12	verification/src/error.rs:247:impl fmt::Display for BlockError {
14:    14	error/src/lib.rs:30:impl fmt::Display for Error {
15:    15	error/src/internal.rs:35:impl fmt::Display for InternalError {
```

Reply:
I think I should use `{:#}` this way to express it instead of directly modifying it.

Other internal errors don't need to be modified, but here, using `{:#}` returns an RPC error